packages/webui/src/client/ui/PreviewPopUp/PreviewPopUp.scss

```
@@ -433,4 +432,8 @@
 			font-weight: 300;
 		}
 	}
+
+	* + :last-child:not(.preview-popUp__box-layout):not(.preview-popUp__iframe):not(.preview-popUp__video) {
```

Stylelint (17.5.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)